### DISCUSSION TITLE
Disable auto-merge for .0 releases

### DISCUSSION THREAD

**Discussion by ahoj-k8s** [renovatebot/renovate#12746] [Request Help]
### What would you like Renovate to be able to do?

For our current project we would like to disable auto-merges for fresh minor releases, meaning `.0` releases. Experience shows lots of `.0` releases are error prone. Thus we want to wait until the next release after the `.0` one. So do not install automatically if there is a patch .0 release, just create a MR.

**Example**
Don't auto-merge if current version is 1.25.5 and latest version is 1.26.0, but auto-merge when latest is 1.26.1 or later.

### If you have any ideas on how this should be implemented, please tell us here.

no

### Is this a feature you are interested in implementing yourself?

Yes

**Comment by rarkins** [comment]
This is too narrow of a use case for us to dedicate an entire feature too, but if we had a `packageRules` option `matchNewVersion` which functions similarly to `matchCurrentVersion` (https://github.com/renovatebot/renovate/issues/12742) then you could disable automerge that way. Feel free to create a new feature request for that more generic approach. I have converted this one to a discussion.

**Comment by viceice** [comment]
I think this can be done with `allowedVersions`  and a regex

https://docs.renovatebot.com/configuration-options/#allowedversions

```json
{
  "packageRules": [
    {
      "matchPackageNames": ["com.thoughtworks.xstream:xstream"],
      "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[1-9][0-9]*$/"
    }
  ]
}
```

**Reply by rarkins** [reply]
won't that mean .0 releases are *blocked*?

**Reply by viceice** [reply]
oh yes. 🤷‍♂️

**Comment by ohartl** [comment]
> I think this can be done with `allowedVersions` and a regex
> 
> https://docs.renovatebot.com/configuration-options/#allowedversions
> 
> ```json
> {
>   "packageRules": [
>     {
>       "matchPackageNames": ["com.thoughtworks.xstream:xstream"],
>       "allowedVersions": "/^[0-9]+\\.[0-9]+\\.[1-9][0-9]*$/"
>     }
>   ]
> }
> ```

This could work, but he wants to apply this filter for every package. Is it possible to put something else than a list of package names? Like a pattern?

**Reply by rarkins** [reply]
matchPackagePatterns
